src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -63,27 +63,6 @@
     end
   end
 
-  describe '#repo_status_icon' do
```

Comment:
This helper is still present, so we should update the specs instead of removing them.

Reply:
Bento helper `repo_status_icon` and its tests were removed in 1a0eabefd1aab039f61e43415eea1a578e904393. Newer helper `webui2_repo_status_icon` was renamed to `repo_status_icon` in ef35b3ab2b06d2c53bf8a4ec7495956372225f13. That is why "the helper is still present". Do we really need a test for actual renamed to `repo_status_icon` helper, that has this code?:
```
  def repo_status_icon(status)
    REPO_STATUS_ICONS[status] || 'eye'
  end
```

Reply:
It's true that it's pretty basic code... I guess it's fine.